xbmc/windowing/XkbcommonKeymap.cpp

```diff
@@ -0,0 +1,351 @@
+/*
```

Comment:
this file is only used by wayland. hence it should be placed into wayland folder